src/add-api/index.md

```diff
@@ -0,0 +1,31 @@
+# How do I add scoreboards and trophies to my game?
+
+First, you need to implement the Game API in your game's code. Then, you can add scoreboards and trophies to your game.
+
+## Scoreboards
+
+You can add multiple scoreboards to a game, so you could have, for example, one for each level of the game, as well as a global scoreboard. Or, you could make one that ranks players by something besides points, like total time or number of jumps. You have control over sorting options, so you could make a traditional scoreboard, where the same person can occupy multiple slots, or you could make it so that only someone's best score is displayed. You can enable guest scoring, so players don't need an account to submit a score. You can even attach extra, hidden data to scores (for verification, tracking, or other uses).
+
+## Trophies
+
+Trophies come in bronze, silver, gold, and platinum; the material corresponds to how easy or difficult they are to earn. You can give your trophies custom art and clever titles. You can make secret trophies that aren't completely revealed until they are won. You can keep trophies hidden to test them out.
```

Comment:
- [ ] Give guidelines on when you should choose bronze, silver, gold and platinum.
- [ ] Mention that gamers get experience from earning trophies. so we should explain that they get 5 exp for bronze, 10 for silver, 15 for gold and 20 for platinum. So devs should think about earning platinum as 4x as hard as earning a bronze trophy.
- [ ] Explain secret trophies more. You can mark  your trophy as a secret trophy which hides the details of the trophy aside from the name and the difficulty level from the user until it is achieved. 
- [ ] don't use secret trophies for all of your trophies but if you want to add in a bonus challenge, you can make it secret.
- [ ] Explain hidden state for trophies. The dev will still see the trophy but gamers will not. It's purpose is to make it easier for the developer to integrate it into their game. Don't forget to unhide!